pkgs/development/rocm-modules/6/rocdbgapi/default.nix

```diff
@@ -12,7 +12,7 @@
   texliveSmall,
   doxygen,
   graphviz,
-  buildDocs ? true,
+  buildDocs ? false, # TODO: revert to true, docs build fixed in PR pending merge
```

Comment:
nit: Which PR?

Reply:
#423435

Has been merged so need to drop this change.